README.md

```diff
@@ -1,702 +1,213 @@
-    ARTPush.didFailToRegisterForRemoteNotificationsWithError(error, rest: rest)
-}
-```
-
-Only one instance of `ARTRest` or `ARTRealtime` at a time must be [activated for receiving push notifications](https://www.ably.com/docs/general/push/activate-subscribe). Having more than one activated instance at a time may have unexpected consequences.
-
-### Location pushes
-
-Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
-
-* To receive location push notifications you need to request another device token specifically for the location pushes purpose. You do so by calling `CLLocationManager.startMonitoringLocationPushes(completion:)` within a `ARTPushRegistererDelegate.didActivateAblyPush(:)` delegate method (which is a callback for the `ARTRealtime.push.activate()` call).
-
-* Once you receive the location push token, save it by calling a new `ARTPush.didRegisterForLocationNotifications(withDeviceToken:realtime:)` method (note the "Location" word in the name of this method). Ably will call the `ARTPushRegistererDelegate.didUpdateAblyPush:` callback on a successful or failed attempt to save the token.
-
-Refer to the [embed APNs example app](https://github.com/ably/ably-cocoa/blob/main/Examples/AblyPush/README.md) on how this can be implemented.
-
-### macOS & tvOS
-
-Be aware that Push Notifications are currently unsupported for macOS and tvOS. You can only use the [Push Admin](https://www.ably.com/docs/general/push/admin) functionalities, for example:
-
-```swift
-let recipient: [String: Any] = [
-    "clientId": "C04BC116-8004-4D78-A71F-8CA3122734DB"
-]
-let data: [String: Any] = [
-    "notification": [
-        "title": "Hello from Ably!",
-        "body": "Example push notification from Ably."
-    ],
-    "data": [
-        "foo": "bar",
-        "baz": "qux"
-    ]
-]
-realtime.push.admin.publish(recipient, data: data) { error in
-    print("Push published:", error ?? "nil")
-}
-```
-
-Available demos: [macOS](https://github.com/ably/demo-macos) and [tvOS](https://github.com/ably/demo-tvos).
-
-## Using the Realtime API
-
-<!--
-NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE N
-**********************************************
-==============================================
-
-(Sorry for the noise.)
-
-Those examples need to be kept in sync with:
+## Supported platforms
 
-Test/Tests/ReadmeExamplesTests.swift
+Ably aims to support a wide range of platforms. If you experience any compatibility issues, open an issue in the repository or contact [Ably support](https://ably.com/support).
 
-==============================================
-**********************************************
-OTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NO
--->
+The following platforms are supported:
 
-### Introduction
+| Platform | Support |
+|----------|---------|
+| iOS| >= 10 |
```

Comment:
These values are wrong; please check against the ones you've removed.

Reply:
No probs. I can re-add the originals. 

My reasoning was this from the `package swift`:

```
let package = Package(
    name: "ably-cocoa",
    platforms: [
        .macOS(.v10_11),
        .iOS(.v9),
        .tvOS(.v10)
```